Box.V2/Request/HttpRequestHandler.cs

```diff
@@ -156,6 +156,10 @@ private HttpRequestMessage BuildMultiPartRequest(BoxMultiPartRequest request)
             var stringParts = request.Parts.Where(p => p.GetType() == typeof(BoxStringFormPart))
                 .Select(p => p as BoxStringFormPart);
 
+            // Create the string part
+            foreach (var sp in stringParts)
```

Comment:
The main reason for this change is that the wrong order of the metadata and file content is causing trouble for backend to support zone, since we need to know the metadata first to decide which zone to use. We decided to upgrade to all SDK's at this moment and deprecate the wrong order case some time.

I haven't tested it for performance impact.